### DISCUSSION TITLE
Post delay on F-Droid

### DISCUSSION THREAD

**Discussion by Korb** [tobexyz/yaacc-code#87] [General]
15 hours ago version 4.0.2 was published on Github. On F-Droid the Suggested version is still 4.0.1 (added on 05/16/2023). Including after a forced update of the repositories. How long does it usually take from the moment a version is published on Github until the release appears on F-Droid?

**Comment by tobexyz** [comment]
hi, the new release will be automatically discovered, scanned, build and published by f-droid. Normally this process takes a few days.
